templates/image/partial/list.html.twig

```diff
@@ -5,7 +5,7 @@
 {% for row in 0 .. (rows - 1) %}
     <div class="row">
         {% for col in 0 .. (cols - 1) %}
-            {% set index = (row * cols + col)|number_format %}
+            {% set index = row * cols + col %}
```

Comment:
The lack of `|number_format` caused issues in the past. I'm not requesting a change, just pointing out something we need to keep an eye on.

Reply:
I didn't do any work on the image index or image_show (since I didn't think it was worth it since those views are just administrative anyway), so I think the lack of number_format is from an old version of the file. I'll update it to use number_format for safety's sake, though.